angular/projects/admin/src/app/admin/settings/settings-dialog-manage-languages/settings-dialog-manage-languages.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+  <div class="col">
+  </div>
+</div>
+<div class="row">
+  <div class="col">
+    <form [formGroup]="languagesForm"
+          novalidate>
+      <div formArrayName="languages">
+        <div class="row"
+             *ngFor="let language of languagesFormArray.controls; let i=index"
+             [formGroupName]="i">
+          <div class="col">
+            <mat-form-field>
+              <input matInput
+                     placeholder="Language"
+                     formControlName="name">
```

Comment:
Make into a `<select>` form input

Reply:
Solved on this commit https://github.com/oddbit/tanam/pull/51/commits/326cd1c894f17e29e2191d6e9a1d64fd8b14ddf5#diff-a84485af018db7b862896327b91e7f44R15